utilities/titandb/db_impl.cc

```diff
@@ -295,6 +295,11 @@ Status TitanDBImpl::DropColumnFamilies(
 
   MutexLock l(&mutex_);
 
+  // TODO:
+  // As rocksdb described, `DropColumnFamilies()` only records the drop the column family specified by ColumnFamilyHandle.
```

Comment:
`only records the drop the column family`, meaning is unclear, probablly a typo.